app/contracts/BankV1.spec.ts

```
@@ -0,0 +1,69 @@
+let bank = BankV1.withAddress(createAddress(0).toString())
+
+const mohamed = createAddress(420420)
+const ahmed = createAddress(6969)
+const asmaa = createAddress(1111)
+const attacker = createAddress(1234567890)
+
+beforeEach(async () => {
+  client = createMemoryClient()
+
+  const { createdAddress } = await client.tevmDeploy(BankV1.deploy())
+  bank = BankV1.withAddress(createdAddress!)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)